src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php

```diff
@@ -146,20 +160,43 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $bundle = $kernel->getBundle($input->getArgument('bundle'));
                 $transPaths = array($bundle->getPath().'/Resources/translations');
                 if ($this->defaultTransPath) {
-                    $transPaths[] = $this->defaultTransPath.'/'.$bundle->getName();
+                    $transPaths[] = $this->defaultTransPath.'/bundles/'.$bundle->getName();
+                }
```

Comment:
After reading this doc https://symfony.com/doc/4.0/bundles/override.html#translations and do some tests I came to the conclusion that adding this paths is really not neccessary as `$this->defaultTransPath` sub-directories are discovered anyway. So we can remove them safely (i.e. all about overriding bundle translations under `$this->defaultTransPath`).

Reply:
done, thanks!